web/src/setup.ts

```diff
@@ -26,10 +26,10 @@ $(() => {
         return this[0].getBoundingClientRect();
     };
 
-    $.fn.expectOne = function () {
+    $.fn.expectOne = function <T>() {
         if (blueslip && this.length !== 1) {
             blueslip.error("Expected one element in jQuery set", {length: this.length});
         }
-        return this;
+        return this as JQuery<T>;
```

Comment:
This is not safe. It converts from `JQuery<A>` to `JQuery<B>` without any checks. Do not do this. If you think you need this, you have a bug somewhere else.